src/application-base/private/translationsLoaders.js

```diff
@@ -0,0 +1,62 @@
+const loadTranslationsFile = (locale) => {
+  switch (locale) {
+    case 'en':
+      return import(/* webpackChunkName: "en-translations" */ 'en.js');
```

Comment:
could we use "dynamic expressions" to load any aggregated translation in the aggregate translations folder? this is similar to how you load things in the workflow server.

https://webpack.js.org/api/module-methods/#dynamic-expressions-in-import

Reply:
Thinking more about this, i don't think it's nearly as easy as i want it to be due to how we're resolving the translations currently, you wouldn't sanely do this `import({locale}.js);` because it would load in way to many files (i think). To get around that we'd have to mess with our webpack config in one way or another which gets us into non passive territory pretty quick.

Reply:
Yeah, that was my original thinking when I first making the changes. But it looks like the magic comment: `/* webpackMode: "lazy" */` might actually solve this issue.